drbd_metrics_test.go

```diff
@@ -181,6 +181,6 @@ func TestNewDrbdCollectorChecksDrbdsetupExecutableBits(t *testing.T) {
 func TestDRBDCollector(t *testing.T) {
 	clock = StoppedClock{}
 
-	collector, _ := NewDrbdCollector("test/fake_drbdsetup.sh")
+	collector, _ := NewDrbdCollector("test/fake_drbdsetup.sh", "splitbrainfake")
 	expectMetrics(t, collector, "drbd.metrics")
 }
```

Comment:
what about adding a dedicated test case? it should be easy enough now that the dedicated method is extracted.

Reply:
unfortunately isn't that easy enough :grin: 

Since the metrics aren't data but directory/files we need to create and remove.

Reply:
done